lib/client.js

```diff
@@ -123,18 +125,19 @@ Client.prototype._invoke = function(method, arguments, location, callback, optio
         assert.ok(!style || style == 'document', 'invalid message definition for rpc style binding');
         message = self.wsdl.objectToDocumentXML(input.$name, arguments, input.targetNSAlias, input.targetNamespace);
     }
-    xml = "<soap:Envelope " + 
-            "xmlns:soap=\"http://schemas.xmlsoap.org/soap/envelope/\" " +
+    
+    xml = "<"+soapHeader+":Envelope " + 
```

Comment:
Short answer:  I think the change is needed to conform to a customer's server/backend convention.